assets/js/theme/global/modal.js

```diff
@@ -196,12 +210,65 @@ export class Modal {
         this.$content.html('');
     }
 
+    setupFocusableElements(modalType) {
+        this.$lastPageFocusedElement = $(document.activeElement);
+
+        const $collection = focusableElements[modalType]();
+        $collection.each((index, element) => $(element).removeClass(focusedClass)); 
+        const $elementToFocus = $($collection.get(0));
+        $elementToFocus.focus().addClass(focusedClass);
+
+        $('#modal').on('keydown', this.onTabbing.bind(null, modalType));
+    }
+
+    onTabbing(modalType, event) {
+        const isTAB = event.which === TAB;
+
+        if (!isTAB) return;
+
+        const $collection = focusableElements[modalType]();
+        const collectionLastIdx = $collection.length - 1
+
+        const $currentElement =  $collection.filter(`.${focusedClass}`);
+        const currentElementIdx = $collection.index($currentElement);
+
+        const direction = event.which === TAB && event.shiftKey ? 'backwards' : 'forwards';
```

Comment:
Direction var could be changed to boolean type

Reply:
If you don't mind I will leave it in such way. Suppose that it is little bit more readable